app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -417,7 +418,8 @@ function AcuantCapture(
       if (forceNativeCamera) {
         trackEvent('IdV: Native camera forced after failed attempts', {
           field: name,
-          failed_attempts: failedCaptureAttempts,
+          failed_capture_attempts: failedCaptureAttempts,
+          failed_submission_attempts: failedSubmissionAttempts,
```

Comment:
Heads-up if #6927 is merged before this, you'll need to rebase and update the [new analytics method signature](https://github.com/18F/identity-idp/pull/6927/files#diff-d648e6f0a371a7ec4e80299f0a2a000d21e9789ad6601fb8351e7101b346e352).

Reply:
Thanks I'll do that here in a bit